common/src/main/java/org/opensearch/ml/common/CommonValue.java

```diff
@@ -222,6 +225,15 @@ public class CommonValue {
             + MODEL_MAX_LENGTH_FIELD + "\":{\"type\":\"integer\"},\""
             + ALL_CONFIG_FIELD + "\":{\"type\":\"text\"}}},\n"
             + "      \""
+            + MLModel.IS_ENABLED_FIELD
+            + "\" : {\"type\": \"boolean\"},\n"
+            + "      \""
+            + MLModel.IS_MODEL_CONTROLLER_ENABLED_FIELD
+            + "\" : {\"type\": \"boolean\"},\n"
+            + "      \""
+            + MLModel.MODEL_RATE_LIMITER_CONFIG_FIELD
+            + "\" : {\"type\": \"flat_object\"},\n"
+            + "      \""
```

Comment:
Is it possible to combine these fields into 1 field saving some spaces? for example, if this rate limiter config field is empty, we can treat this model as controller disabled, etc.

Reply:
This is not possible because this field is used as a flag to notify model if we need to auto-deploy/undeploy its corresponding controller.